docs/writing-tests/assumptions.md

```diff
@@ -11,7 +11,6 @@ tests can freely rely on these assumptions being true:
  * The initial value of `color` is `black`.
  * The user stylesheet is empty (except where indicated by the tests).
  * The device is interactive and uses scroll bars.
- * The device has the Ahem font installed.
```

Comment:
This "assumption" is needed until part 2.

Reply:
True, to be done as a final cleanup PR